ray-operator/test/sampleyaml/raycluster_test.go

```diff
@@ -121,3 +121,55 @@ func TestRayCluster(t *testing.T) {
 		})
 	}
 }
+
+func TestRayClusterSuspend(t *testing.T) {
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "ray-cluster.complete.yaml",
```

Comment:
Construct the RayCluster with ApplyConfiguration like [this](https://github.com/ray-project/kuberay/blob/9321b2d33c6b92910c3ae0fc34933093d3e0ee7c/ray-operator/test/e2e/rayjob_suspend_test.go#L33). I prefer to avoid relying on external YAML files as much as possible.

Reply:
Hi @kevin85421, this is ready.